public/app/features/explore/ExploreQueryInspector.tsx

```diff
@@ -54,8 +54,8 @@ export function ExploreQueryInspector(props: Props) {
   };
 
   const queryInspectorTab: TabConfig = {
-    label: 'Query',
-    value: 'query',
+    label: 'Query Inspector',
```

Comment:
```suggestion
    label: 'Query inspector',
```

Reply:
I will change it to just `Query` as that is how it is done in Panel inspect and that is used across documentation. Just need to create separate PR so it can be included in 8 release. :)

Reply:
Sounds good. My comment is non-blocking, and it is best that I stop potential stepping on Jita’s toes.

Reply:
But still, it was very good point!